yuuna/lib/src/pages/implementations/subtitle_options_dialog_page.dart

```diff
@@ -459,4 +567,53 @@ class _SubtitleOptionsDialogPage
   void executeSet() async {
     await setValues(saveOptions: false);
   }
+
+  /// Pick a font file with a built-in file picker.
+  Future<bool> pickFontFile() async {
+    FilePickerResult? result = await FilePicker.platform.pickFiles(
+      type: FileType.custom,
+      allowedExtensions: ['ttf', 'otf'],
+    );
+    if (result != null) {
+      _fontNameController.text = result.files.single.name.split('.').first;
+      var custom = FontLoader(_fontNameController.text);
+      custom.addFont(loadFont(result.files.single.path ?? ''));
+      await custom.load();
+      return true;
+    }
+    return false;
+  }
+
+  Future<ByteData> loadFont(String path) async {
+    File file = File(path);
+    Uint8List bytes = await file.readAsBytes();
+    return ByteData.view(bytes.buffer);
+  }
+
+  void showColorPicker(String target) {
```

Comment:
Is it possible you could change this to use an enum or a separate function instead of taking a `String` parameter?